doc/design.txt

```diff
@@ -280,10 +283,7 @@ higher priority when testing the image trailers.
             image-ok | Any    | Unset  |
            copy-done | Any    | Any    |
     -----------------+--------+--------'
-             pending |        |   T    |
-          confirmed  |   X    |        |
-    -----------------+--------+--------'
-     swap: test                        |
+     result: BOOT_SWAP_TYPE_TEST       |
```

Comment:
I changed the same thing on my last doc updates, but decided to rollback before finishing the commit! :-)